### ISSUE TITLE
can't load vanilla world from latest version

### ISSUE THREAD

**Issue by bbSnavy** [macmv/bamboo#13]
trying to load a vanilla world from 1.19 just does nothing

**Comment by macmv** [comment]
yeah, this is expected. It's because they switched region formats (the .mca to .mcr file change), so the parser just finds no valid region files.
